dissect/target/plugins/os/unix/config.py

```diff
@@ -81,54 +102,193 @@ def parse_file(self, fh: io.TextIO) -> None:
         self.parsed_data.read_file(open_file)
 
 
-class Txt(LinuxConfigurationParser):
+class Txt(ConfigurationParser):
     def parse_file(self, fh: TextIO) -> None:
         self.parsed_data = {"content": fh.read(), "size": str(fh.tell())}
 
 
-class Default(LinuxConfigurationParser):
-    EMPTY_SPACE = re.compile(r"\s+")
+class Default(ConfigurationParser):
+    def __init__(
+        self,
+        collapse: Optional[Union[bool, set]] = False,
+        seperator: tuple[str] = (r"\s",),
+        comment_prefixes: tuple[str] = (";", "#"),
+    ) -> None:
+        super().__init__(collapse, seperator, comment_prefixes)
+        self.SEPERATOR = re.compile(rf"\s*?[{''.join(seperator)}]\s*?")
+        self.COMMENTS = re.compile(rf"\s*[{''.join(comment_prefixes)}]")
 
     def parse_file(self, fh: TextIO) -> None:
-        new_info = {}
+        information_dict = {}
+
+        skip_lines = self.comment_prefixes + ("\n",)
+        prev_key = None, None
```

Comment:
```suggestion
        prev_key = (None, None)
```

Nicer to have this tuple explicit, but should it be a tuple in the first place?